src/components/simplified-exchange-component.review-fixes.ts

```
@@ -0,0 +1,171 @@
+proto._populateMaxBalance = function populateMaxBalance(this: ExchangeInternals, force: boolean): boolean {
+  if (this._state.direction === "deposit" && !this._getSelectedToken()) {
+    return false;
+  }
+
+  return originalPopulateMaxBalance.call(this, force);
+};
+
+proto._handleTransactionSuccess = function handleTransactionSuccess(this: ExchangeInternals): void {
+  const direction = this._state.direction === "deposit" ? "Bought" : "Sold";
+  const selectedToken = this._getSelectedToken();
+  const tokenSymbol = this._state.direction === "deposit" ? selectedToken?.symbol ?? "token" : INVENTORY_TOKENS.UUSD.symbol;
```

ESLint

[error] Replace `selectedToken?.symbol·??·"token"` with `(selectedToken?.symbol·??·"token")`

(prettier/prettier)